person.rb

```diff
@@ -29,6 +30,14 @@ def correct_name
   def of_age?
     @age >= 18
   end
+
+  def add_rental(rental)
+    @rentals << rental
```

Comment:
- [ ] here you have to create a new instance of Rentals, remember in your rental you are already pushing to person.rental, so your code should look like this `Rental.new(date, book, self)`